addons/portal_rating/controllers/thread.py

```diff
@@ -0,0 +1,14 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.fields import Domain
+from odoo.addons.mail.controllers import thread
+
+
+class ThreadController(thread.ThreadController):
+    def _get_fetch_domain(self, thread, **kwargs):
+        domain = super()._get_fetch_domain(thread, **kwargs)
+        if kwargs.get("rating_value"):
```

Comment:
This might prevent from searching 0, which is probably why the previous code had `, False) is not False`.